test/resource.test.js

```diff
@@ -0,0 +1,373 @@
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La lección no existe");
+            done();
+        });
+    });
+
+    it("POST /Resource Lesson Bad Request File", (done) =>{
+        chai.request(server).post("/resources/lesson")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idLesson', '6171fb2eeb326a2f1850c22e')
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("Necesita subir una imagen");
+            done();
+        });
+    });
+
+    it("POST /Resource Lesson Bad Request File Extension", (done) =>{
+        chai.request(server).post("/resources/lesson")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idLesson', '6171fb2eeb326a2f1850c22e')
+        .attach('file', `${__dirname}/resources/imageError.bmp`, 'imageError.bmp')
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("Extensión no admitida o el archivo debe pesar menos de 10MB");
+            done();
+        });
+    });
+
+    it("POST /Resource Lesson", (done) =>{
+        chai.request(server).post("/resources/lesson")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idLesson', '6171fb2eeb326a2f1850c22e')
+        .attach('file', `${__dirname}/resources/image.png`, 'image.png')
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(201);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La imagen se guardo exitosamente");
+            done();
+        });
+    });
+
+    after((done) => {
+        chai.request(server).delete("/resources")
+        .send({
+            URL: "/images/lessons/6171fb2eeb326a2f1850c22e.png"
+        })
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            done();
+        });
+    });
+});
+
+describe("POST Tests Resource Audio",()=>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("POST /Resource Audio Bad Request", (done) =>{
+        chai.request(server).post("/resources/audio")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idQuestion', '6164vrfdb5')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
        .field("idQuestion", '6164vrfdb5')
```